<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDistributorForm</class>
 <widget class="QFrame" name="CDistributorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>178</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Distributor editor</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="hl_DistributorEditor">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Distributor">
     <property name="title">
      <string>Distributor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="BlackGui::CLabelAndIcon" name="lai_Id">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="5">
       <widget class="BlackGui::Editors::CValidationIndicator" name="val_Indicator">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_Alias1">
        <property name="placeholderText">
         <string>An alias</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="BlackGui::Components::CDbDistributorSelectorComponent" name="distributor_Selector">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_Alias2">
        <property name="placeholderText">
         <string>Another alias</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_Alias">
        <property name="text">
         <string>Alias1/2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_Description">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Timestamp">
        <property name="text">
         <string>Last updated</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="BlackGui::CDropSite" name="drop_DropData">
        <property name="text">
         <string>Drop data here</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="le_Description">
        <property name="placeholderText">
         <string>Describe distributor</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="le_Updated">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CDropSite</class>
   <extends>QLabel</extends>
   <header>blackgui/dropsite.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::CLabelAndIcon</class>
   <extends>QFrame</extends>
   <header>blackgui/labelandicon.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CValidationIndicator</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/validationindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbDistributorSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbdistributorselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Description</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
